src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -432,18 +431,21 @@ const submitEncryptModeForm: ControllerHandler<
     const metadata: StripePaymentMetadataDto = {
       formTitle: form.title,
       formId,
+      submissionId: pendingSubmissionId,
       paymentId,
       paymentContactEmail: paymentReceiptEmail,
     }
 
+    const paymentReceiptDescription = `${form.payments_field.description}\nFormSG form: ${form.title}\nResponse ID: ${pendingSubmissionId}`
```

Comment:
/nit Would this make the description content clearer?
```suggestion
    const paymentReceiptDescription = [
      form.payments_field.description,
      `FormSG form: ${form.title}`,
      `Response ID: ${pendingSubmissionId}`
    ].join("\n")
```

Reply:
omg smartttt